doc/nrf/migration/migration_guide_1.x_to_2.x.rst

```diff
@@ -0,0 +1,432 @@
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_half_float`          | no equivalent                     |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_float`               | `zcbor_float32_put`               |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_double`              | `zcbor_float64_put`               |
+    |                                   | `zcbor_double_put`                |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_text_string`         | `zcbor_tstr_encode`               |
+    |                                   | `zcbor_tstr_encode_ptr`           |
+    |                                   | `zcbor_tstr_put_lit`              |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_text_stringz`        | `zcbor_tstr_put_term`             |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_byte_string`         | `zcbor_bstr_encode`               |
+    |                                   | `zcbor_bstr_encode_ptr`           |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_null`                | `zcbor_nil_put`                   |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_undefined`           | `zcbor_undefined_put`             |
+    +-----------------------------------+-----------------------------------+
+
+There is no direct replacement for the `cbor_value_is_null` and `cbor_value_is_undefined` functions in zcbor.
+To check whether such value is where expected, you can utilize the following code:
+
+.. code-block:: c
+
+   bool ok;
+   ...
+   /* Make sure that not in error state first */
+   if (zs->constant_state->error != 0) {
+       /* Already in error */
+       ...
+   }
+   ok = zcbor_nil_decode(zs, NULL);
+   if (!ok && zs->constant_state->error == ZCBOR_ERR_WRONG_TYPE) {
+       /* Process non NULL element */
+       zcbor_pop_error(zs);
+       ...
+   } else if (!ok) {
+       /* Other error */
+       ...
+   }
+   /* Successfully decoded NULL element in CBOR stream */
+   ...
+
+.. _migration_tinycbor_to_zcbor_mcumgr:
+
+MCUmgr commands and groups migration
+====================================
+
+In contrast to :ref:`migration_tinycbor_to_zcbor_general`, except for tuning some Kconfig configuration constants, there is no need for zcbor setup as the MCUmgr library takes case of the task.
+Most of the task which consists of function replacement is actually covered by the :ref:`migration_tinycbor_to_zcbor_general`, with exception to usage of `cborattr` which is covered by paragraph :ref:`migration_tinycbor_to_zcbor_mcumgr_cborattr`.
```

Comment:
```suggestion
In contrast to :ref:`migration_tinycbor_to_zcbor_general`, there is no need for zcbor setup for the MCUmgr commands and groups, because the MCUmgr library takes case of the task.
The only exception is tuning some Kconfig configuration constants.

Most of the function replacement task is covered by the :ref:`migration_tinycbor_to_zcbor_general`, with an exception for the `cborattr` usage, which is covered by a paragraph under :ref:`migration_tinycbor_to_zcbor_mcumgr_cborattr`.
```